src/Agda2Hs/Compile/Term.hs

```diff
@@ -180,6 +181,7 @@ lambdaCase q es = setCurrentRange (nameBindingSite $ qnameName q) $ do
     -- simply return the body.
     [Hs.Match _ _ [] (Hs.UnGuardedRhs _ rhs) _] -> return rhs
     _ -> do
+      incrementLCase
       alts <- mapM clauseToAlt cs -- Pattern lambdas cannot have where blocks
       args <- compileElims rest
       return $ eApp (Hs.LCase () alts) args
```

Comment:
Could you use the `hsLCase` defined on line 214 of `Compile.Types` so that its comment holds?